### DISCUSSION TITLE
Lagging,fps drop and jump.

### DISCUSSION THREAD

**Discussion by Ryuu470** [ppy/osu#29652] [Q&A]
### Type

Crash to desktop

### Bug description

so i was playing stable version of osu and wanted to try osu lazer because i heard that i can get lower delay and after a few minutes, the fps start dropping

### Screenshots or videos

https://github.com/user-attachments/assets/d084b3af-fd43-4a7c-b8a9-06a70b801bf7

### Version

osu!lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16811195/compressed-logs.zip)

**Comment by peppy** [comment]
Probably pc doing something in the background.

**Reply by Ryuu470** [reply]
i was thinking the same so i opened osu stable version and somehow the fps and the gameplay have no issues

**Reply by Ryuu470** [reply]
update : i reopened osu lazer again after checking the osu stable and the fps still dropping

**Comment by smoogipoo** [comment]
Take a video with the frame time graphs shown by pressing Ctrl+F11 twice.

**Reply by Ryuu470** [reply]
recently, its back to normal. i will send the video it happens again.

**Reply by Ryuu470** [reply]
https://github.com/user-attachments/assets/a05b47cc-4e87-4812-99d9-3a22f2cdf70a

sorry for make you waiting

**Reply by peppy** [reply]
Are you sure your PC isn't overheating?

**Reply by Ryuu470** [reply]
![image](https://github.com/user-attachments/assets/2aaa77dd-8b2e-46e3-9140-4fcecd2c2429)
maybe no 
edit: (i dont know normal temp for laptop xD)
